specification/digitaltwins/data-plane/Microsoft.DigitalTwins/preview/2020-05-31-preview/digitaltwins.json

```diff
@@ -0,0 +1,1497 @@
+        "x-ms-examples": {
+          "GetComponent": {
+            "$ref": "./examples/GetComponent.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/digitalTwinId"
+          },
+          {
+            "$ref": "#/parameters/componentPath"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Weak Etag.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Twins"
+        ],
+        "operationId": "DigitalTwins_UpdateComponent",
+        "description": "Updates a component on a digital twin.\nStatus codes:\n200 (OK): Success.\n400 (Bad Request): The request is invalid.\n404 (Not Found): There is either no digital twin with the provided id or the component path is invalid.",
+        "x-ms-examples": {
+          "PatchComponentBasicExample": {
+            "$ref": "./examples/PatchComponentBasicExample.json"
+          },
+          "PatchComponentAdvancedExample": {
+            "$ref": "./examples/PatchComponentAdvancedExample.json"
+          }
+        },
+        "consumes": [
+          "application/json-patch+json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/digitalTwinId"
+          },
+          {
+            "$ref": "#/parameters/componentPath"
+          },
+          {
+            "name": "patchDocument",
+            "description": "An update specification described by JSON Patch. Updates to property values and $model elements may happen in the same request. Operations are limited to add, replace and remove.",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object"
+              }
+            }
+          },
+          {
+            "$ref": "#/parameters/if-match"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success",
+            "headers": {
+              "ETag": {
+                "description": "Weak Etag.",
+                "type": "string"
+              }
+            }
+          },
+          "202": {
+            "description": "Asynchronous Success"
```

Comment:
LRO?

Reply:
Similar to the above comment. In the case when the digital twin represents a device that may or may not be connected, this API returns 202 and the client uses the digital twin notification feed to get updates.